iceoryx_posh/source/roudi/roudi.cpp

```diff
@@ -235,6 +239,21 @@ void RouDi::processMessage(const runtime::MqMessage& message,
         }
         break;
     }
+    case runtime::MqMessageType::CREATE_EVENT_VARIABLE:
+    {
+        if (message.getNumberOfElements() != 2)
+        {
+            LogError() << "Wrong number of parameters for \"MqMessageType::CREATE_EVENT_VARIABLE\" from \""
+                       << processName << "\"received!";
+            errorHandler(
+                Error::kPORT_MANAGER__INTROSPECTION_MEMORY_MANAGER_UNAVAILABLE, nullptr, iox::ErrorLevel::MODERATE);
```

Comment:
c&p error. This is not true. Please remove it from here and also from the place you copied it. This is actually something the application has to be notified about, since there is nothing RouDi can do. This will be tackled by #332

Reply:
done